Microsoft.Toolkit.Uwp/Helpers/ConnectionHelper.cs

```diff
@@ -106,5 +109,54 @@ public static ConnectionType ConnectionType
                 return connectionType;
             }
         }
+
+    /// <summary>
+    /// Gets connection ssid for the current Wifi Connection.
+    /// </summary>
+    /// <returns> string value of current ssid/></returns>
+    ///
+    public static async Task<string> GetNetworkSsidAsync()
```

Comment:
no i meant that GetNetworkSsid is very limited in functionality. If we add something it should be broad enough to cater for any network type.

Having said that we are discussing use of user voice to see what community wants. We can put exposing GetNetworkName with optional filter there and if there's enough interest, we can start work on this again. Until then maybe we should stop work on this PR

Apologies,

Hermit